cmake/modules/SwiftHandleGybSources.cmake

```diff
@@ -42,6 +42,10 @@ function(handle_gyb_source_single dependency_out_var_name)
       ${GYB_SINGLE_FLAGS})
 
   set(gyb_tool "${SWIFT_SOURCE_DIR}/utils/gyb")
+
+  # GYB needs to be run via Python
+  set(gyb_tool_command "python" ${gyb_tool})
```

Comment:
@hughbe that works by allowing you to specify the path to the python interpreter at the cmake configuration time (i.e. -DPYTHON_EXECUTABLE=/usr/x86_64-pc-linux-gnu/bin/python2.7).